app/uk/gov/hmrc/economiccrimelevyregistration/models/errors/KnownFactsError.scala

```diff
@@ -14,13 +14,12 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.economiccrimelevyregistration.utils
+package uk.gov.hmrc.economiccrimelevyregistration.models.errors
 
-import java.util.UUID
-import javax.inject.Singleton
+trait KnownFactsError
```

Comment:
doesn't look like the trait is used, can be removed?

Reply:
Hmm, it is used in couple of classes and error cases.